frame/lending/Cargo.toml

```diff
@@ -42,22 +42,22 @@ plotters = { version = "0.3.1", optional = true }
 scale-info = { version = "1.0", default-features = false, features = [
   "derive",
 ] }
+serde = { version = "1.0.130", features = ["derive"], default-features = false }
 
 [dev-dependencies]
-composable-tests-helpers = { path = "../composable-tests-helpers" }
-frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16" }
+composable-tests-helpers = { path = "../composable-tests-helpers", default-features = false }
+frame-benchmarking = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.16" }
 hex-literal = "0.3.3"
 once_cell = "1.8.0"
 orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", rev = "2b1c9fb367ccb8e13601b2da43d1c5d9737b93c6" }
 orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", rev = "2b1c9fb367ccb8e13601b2da43d1c5d9737b93c6" }
-pallet-assets = { path = "../assets" }
+pallet-assets = { path = '../assets', default-features = false }
```

Comment:
Please use consistent style and avoid adding `default-features = false` unless required.

Reply:
just to be clear, what format is to be used for the string style ?